README.md

```diff
@@ -67,9 +67,10 @@ First install pipenv and create a virtual environment with the needed packages:
 
     To deactivate your virtual environment: ``` deactivate ```
 
-Then launch the API using hypercorn (from the root folder):
+Then launch the API using hypercorn (from the root folder and your virtual environment is activated):
 
 ```sh
+python3 initialize.py
```

Comment:
add explanation on the initiajize script before

Reply:
done